### DISCUSSION TITLE
Docker image version mismatch?

### DISCUSSION THREAD

**Discussion by BenSaus** [strapi/strapi#8576] [Legacy Topics]
I built a docker image file (of my strapi project) from strapi/base:alpine and got the following message in the browser console when running the admin panel:
```
It seems that the built version 3.1.6 is different than your project's one (3.2.5)
```
So when are the docker images updated? 
Should I downgrade or can I build an updated base image? Thanks

Edit: I did an npm audit fix without thinking about it and I'm guessing this is the actual problem

package.json
```
  "dependencies": {
    "knex": "<0.20.0",
    "pg": "^8.4.2",
    "sqlite3": "latest",
    "strapi": "^3.2.5",
    "strapi-admin": "3.1.6",
    "strapi-connector-bookshelf": "3.1.6",
    "strapi-plugin-content-manager": "3.1.6",
    "strapi-plugin-content-type-builder": "3.1.6",
    "strapi-plugin-email": "3.1.6",
    "strapi-plugin-graphql": "^3.1.6",
    "strapi-plugin-upload": "^3.2.5",
    "strapi-plugin-users-permissions": "3.1.6",
    "strapi-utils": "3.1.6"
  },
```
Edit 2: Yep

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
